### PR TITLE
Import SDK assets from the `assets/` directory

### PR DESCRIPTION
**Pull request by MadLittleMods**
Import SDK assets from the `assets/` directory

> Will be easier towards the future when adding more assets. Probably best to keep style.css for now for backwards compat though.
>
> *-- @bwindels, https://github.com/vector-im/hydrogen-web/pull/693#discussion_r853844282*

---

```sh
$ yarn test:sdk
...
SDK works in Vite build ✅
SDK works in CommonJS ✅
```

### CODE REVIEW

File: scripts/sdk/base-manifest.json [hunk 1]
```diff
@@ -10,8 +10,6 @@
         },
         "./paths/vite": "./paths/vite.js",
         "./style.css": "./asset-build/assets/theme-element-light.css",
-        "./theme-element-light.css": "./asset-build/assets/theme-element-light.css",
```
Comment (bwindels) [inline scripts/sdk/base-manifest.json]: Could we keep these for backwards compatibility? Do keep the docs updated though.

**Review by bwindels — APPROVED** [review]
lgtm, thanks!

File: scripts/sdk/base-manifest.json (hunk 1 above)
Comment (MadLittleMods) [reply scripts/sdk/base-manifest.json]: Added them back ⏩

**Comment by MadLittleMods** [comment]
Thanks for the review and merge @bwindels 🐒

**Pull request merged**